<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QWidget" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>15</number>
   </property>
   <property name="spacing">
    <number>15</number>
   </property>
   <item row="0" column="1">
    <widget class="QToolButton" name="toolBtnSysterm">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Color:rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>系统管理</string>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/256x256 (115).png</normaloff>:/image/256x256 (115).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="toolBtnPerson">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Color:rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>人员管理</string>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/256x256 (342).png</normaloff>:/image/256x256 (342).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="toolBtnHelp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Color:rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>查看帮助</string>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/256x256 (833).png</normaloff>:/image/256x256 (833).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolBtnShutdown">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Color:rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>设备关机</string>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/shutdown0.ico</normaloff>:/image/shutdown0.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="toolBtnIdentify">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Color:rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>返回识别</string>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/256x256 (1723).png</normaloff>:/image/256x256 (1723).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolBtnDataManager">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Color:rgb(0, 0, 0)</string>
     </property>
     <property name="text">
      <string>数据管理</string>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/256x256 (848).png</normaloff>:/image/256x256 (848).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolBtnPerson</tabstop>
  <tabstop>toolBtnSysterm</tabstop>
 </tabstops>
 <resources>
  <include location="ikembx00.qrc"/>
 </resources>
 <connections/>
</ui>
